tests/clone_test.go

```diff
@@ -189,16 +190,47 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 		}, 3*time.Minute, 3*time.Second).Should(Equal(clonev1alpha1.Succeeded), "clone should finish successfully")
 	}
 
+	startVM := func(vm *virtv1.VirtualMachine) *virtv1.VirtualMachine {
```

Comment:
This is introducing new helpers, so I would suggest to:
- Implement them as regular functions and not closures.
- Consider placing them in a different common place (`libclientset`?). Maybe in the same package the current kubevirt `Client()` reside.
- Consider returning the error over asserting it inside the helper. It is hard to estimate from what stack level it is called, making it challenging to trace it well. Usually an `offset` is set for the assertions to solve that.
- Use `Patch` to avoid retry.

If you change these to a regular function and take care to return the error, we can do the rest as a follow up.

Reply:
So these have no business being in common packages right now but I tried to explain the issue with the existing helpers in the commit message - https://github.com/kubevirt/kubevirt/pull/12023/commits/2654f2c5b628015c4171bd4478593086a5264615.
Maybe we tackle Start/StopVirtualMachine in a follow up since I imagine it's not going to be straightforward.

> Use Patch to avoid retry

Yes that would be ideal but unfortunately the error coming back from patch also needs to be asserted on

Reply:
> > Use Patch to avoid retry
> 
> Yes that would be ideal but unfortunately the error coming back from patch also needs to be asserted on

The error needs to be asserted, sure. But it does not need to retried.

> Maybe we tackle Start/StopVirtualMachine in a follow up since I imagine it's not going to be straightforward.

I am fine with that, this is why I mentioned:
> If you change these to a regular function and take care to return the error, we can do the rest as a follow up.

Is this something you can take care here?

Reply:
> Is this something you can take care here?

Sure - done